core/pkg/server/config.go

```diff
@@ -0,0 +1,208 @@
+package server
+
+import (
+	"encoding/json"
```

Comment:
i think we want to use github.com/segmentio/encoding/json
instead of the default one, if there are place where we use the other one we need to fix it

Reply:
oops, good point! I just let the IDE insert that import for me

Reply:
yeah same, so i think there are places where it is wrong.